openwisp_radius/base/models.py

```diff
@@ -816,11 +816,11 @@ class AbstractRadiusBatch(OrgMixin, TimeStampedEditableModel):
         max_length=20,
         help_text=_('Usernames generated will be of the format [prefix][number]'),
     )
-    pdf = models.FileField(
+    pdf_json = JSONField(
         null=True,
         blank=True,
         verbose_name='PDF',
-        help_text=_('The pdf file containing list of usernames and passwords'),
+        help_text=_('list of usernames and passwords used to create PDF'),
```

Comment:
this is not shown to the user or is it?
If it's not, I would remove the help text and convert it to a comment.

Reply:
Changed to comment.